docs/api-docs/channels/channels-overview.md

```diff
@@ -1,202 +1,199 @@
 # BigCommerce Channels Overview
 
+<!-- https://developer.bigcommerce.com/api-docs/channels/guide/overview -->
+
+## On this page
+
 <div class="otp" id="no-index">
 
-### On this page
-- [Channels Toolkit](#channels-toolkit)
-- [Channel Apps](#channel-apps)
-  - [Discovery](#discovery)
-  - [Display](#display)
-  - [Note](#note)
-  - [Management](#management)
-- [Types of Channels](#types-of-channels)
-  - [Storefront](#storefront)
-  - [Point of Sale (POS)](#point-of-sale-pos)
-  - [Marketplaces](#marketplaces)
-  - [Marketing](#marketing)
-- [Building a Channel App](#building-a-channel-app)
-  - [Getting Started](#getting-started)
-  - [Building the App](#building-the-app)
-- [Related resources](#related-resources)
+- [BigCommerce Channels Overview](#bigcommerce-channels-overview)
```

Comment:
I think the On this page section for this article needs to be reworked. It should not include subheaders such as "Discovery" or "Display".